sample/Waystone.Sample.Infrastructure/DependencyInjection.cs

```diff
@@ -1,15 +1,18 @@
 ﻿namespace Waystone.Sample.Infrastructure;
 
 using Application.WeatherForecasts.Services;
+using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Services;
 
 public static class DependencyInjection
 {
-    public static IServiceCollection AddSampleInfrastructure(this IServiceCollection services)
+    public static IServiceCollection AddSampleInfrastructure(
+        this IServiceCollection services,
+        IConfiguration configuration)
     {
-        services.AddWaystoneInfrastructureBuilder()
-                .AcceptDefaults();
+        services.AddWaystoneInfrastructureBuilder(configuration)
+                .AcceptDefaults(useRedis: true);
```

Comment:
Instead of using a `useRedis` flag, I should probably add another fluent api method that can be chained from `AcceptDefaults` called `WithRedisCaching`